week-8/Homework/mandatory/3-dog-photo-gallery/exercise2.js

```diff
@@ -0,0 +1,25 @@
+function getTheNewPhoto() {
+  fetch("https://dog.ceo/api/breeds/image/random")
+    .then(function (response) {
+      return response.json();
+    })
+    .then(function (data) {
+      var Thescr = data.message;
+      getThePohto(Thescr);
+      console.log(data.message);
+    });
+}
+
+var theBtn = document.getElementById("btn");
+
+function getThePohto(photo) {
```

Comment:
The function is correct. The name could be better like addThePhoto instead of get. Get is usually used when we retrieve something e.g. value from and object for example.